libs/vips/Makefile

```diff
@@ -21,9 +21,11 @@ PKG_CPE_ID:=cpe:/a:vips:vips
 PKG_FIXUP:=autoreconf
```

Comment:
Both not needed with meson

Reply:
Actually all three. Parallel, install, and fix up.

Reply:
Removed.

Reply:
Removed PKG_INSTALL.